src/command_modules/azure-cli-botservice/azure/cli/command_modules/botservice/custom.py

```diff
@@ -63,6 +63,7 @@ def create(cmd, client, resource_group_name, resource_name, kind, description=No
     bot_kind = 'bot'
     webapp_kind = 'webapp'
     function_kind = 'function'
+    show_password = True if not password else False
```

Comment:
Simplify `show_password = not password`

Reply:
@yugangw-msft resolving via comment [below](https://github.com/Azure/azure-cli/pull/8834/files?file-filters%5B%5D=.py#r267631987) and that this comment is now out of date.